### PR TITLE
no pull up anymore

### PR DESCRIPTION
**Pull request by talhaozcan**

### CODE REVIEW

**Comment by talhaozcan** [comment]
:)

**Comment by talhaozcan** [comment]
image

File: src/fooda_bot.py [hunk 1]
```diff
@@ -94,9 +94,9 @@ def gather_food_trucks():
     """These are sorta hard-coded since there's no online source"""
     context = [
         ("Little Blue Bakery", "Teri-yummy", "Zaaki Mediterranean", "Roadie's Diner",),
-        ("Little Blue Bakery", "Kebabish", "Gogi on the Block", "The Bacon Truck"),
+        ("Little Blue Bakery", "Kebabish/JB", "Gogi on the Block", "The Bacon Truck"),
```
Comment (nerdling) [inline src/fooda_bot.py]: What's this?

Comment (talhaozcan) [reply src/fooda_bot.py]: Jeb Bush 😬😬😬just tried :)

**Pull request merged**